### ISSUE TITLE
Spacy 2.2 VersionConflict with pytorch-transformer

### ISSUE THREAD

**Issue by MartinoMensio** [explosion/spacy-transformers#79]
Hi,
I just installed the model `en_pytt_xlnetbasecased_lg-2.1.1` (latest available) over SpaCy 2.2.1 and during the pip installation I received:

```text
ERROR: spacy-pytorch-transformers 0.4.0 has requirement spacy<2.2.0,>=2.1.7, but you'll have spacy 2.2.1 which is incompatible.
```

In the model meta information, the version range for spacy is >=2.1.7 

https://github.com/explosion/spacy-models/blob/7379fc4232f883130ccef29022dd1fd57b3039fb/meta/en_pytt_xlnetbasecased_lg-2.1.1.json#L7

While in the requirements.txt there is also the constraint <2.2.0

https://github.com/explosion/spacy-pytorch-transformers/blob/21a1219f7490bef07ed320fa0bb2e2af3b6a5a61/requirements.txt#L1

I was just wondering if the conflict should exist or not.
If I just ignore the VersionConflict, SpaCy 2.2.1 is still able to load that model and use the sentence embeddings as far as I can see.

Thanks!

**Comment by ines** [comment]
This package isn't "officially" compatible with v2.2 yet (but we're currently working on that and just building the new model packages). So the version contstraint is intentional. It might still work and you can ignore the warnings – we just can't guarantee that everything works as expected.

**Comment by MartinoMensio** [comment]
Thanks! :)

**Comment by ines** [comment]
Just released v0.5.0, which is officially compatible: https://github.com/explosion/spacy-transformers/releases/tag/v0.5.0

Also note that the package is now called `spacy-transformers` 😃
